scene/animation/animation_player.cpp

```diff
@@ -572,7 +572,7 @@ void AnimationPlayer::_animation_process_data(PlaybackData &cd,float p_delta,flo
 
 	} else {
 		
-		if (next_pos<0 or next_pos>len) {
+        if (next_pos<0 || next_pos>len) {
```

Comment:
The fix is good, but you introduced indentation errors :)

Also, no need to mention the files and lines in the commit log, it's a bit redundant with the diff that shows them already.